turf/rhumb_destination/_rhumb_destination.py

```diff
@@ -0,0 +1,118 @@
+from math import fmod
+from typing import List, Dict
+
+import numpy as np
+
+from turf.helpers import convert_length, degrees_to_radians
+from turf.helpers import earth_radius
+from turf.helpers import point, Point
+from turf.invariant import get_coord
+
+
+def rhumb_destination(
+    start: List, distance: float, bearing: float, options: Dict = None
+) -> Point:
+    """
+    Returns the destination {@link Point} having travelled the given distance along a Rhumb line from the
+    origin Point with the (varant) given bearing.
+
+    # https://en.wikipedia.org/wiki/Rhumb_line
+
+    :param origin: origin starting point
+    :param distance: distance distance from the starting point
+    :param bearing: varant bearing angle ranging from -180 to 180 degrees from north
+    :param options: optional parameters
+        [options["units"]={}] can be degrees, radians, miles, or kilometers
+        [options["properties"]={}] Translate GeoJSON Properties to Point
+        [options["id"]={}] Translate GeoJSON Id to Point
+
+    :return: a Feature<Point> Destination point.
+    """
+    kwargs = {}
+    if isinstance(options, dict) and "units" in options:
+        kwargs.update(options)
+
+    if not options:
+        options = {}
+
+    coords = get_coord(start)
+
+    distance_in_meters = convert_length(
+        abs(distance), original_unit=kwargs["units"], final_unit="meters"
+    )
+
+    if distance < 0:
+        distance_in_meters *= -1
+
+    destination = calculate_rhumb_destination(coords, distance_in_meters, bearing)
+
+    # compensate the crossing of the 180th meridian:
+    #  (https://macwright.org/2016/09/26/the-180th-meridian.html)
+    # solution from:
+    # https://github.com/mapbox/mapbox-gl-js/issues/3250#issuecomment-294887678
+
+    if (destination[0] - coords[0]) > 180:
+        destination[0] -= 360
+    elif (coords[0] - destination[0]) > 180:
+        destination[0] += 360
+    else:
+        pass
+
+    return point(destination, options.get("properties", None))
+
+
+def calculate_rhumb_destination(
+    origin: List, distance_in_meters: float, bearing: float, radius: float = None
+) -> List:
+    """
+    Calculates the destination point having travelled along a rhumb line
+    from origin point the given distance on the  given bearing.
+    Adapted from Geodesy:
+    http://www.movable-type.co.uk/scripts/latlong.html#rhumblines
+
+    param origin: point
```

Comment:
perhaps add the expected type in the description: `param origin: point coordinates in [lng, lat] form`